core/src/main/java/io/micronaut/core/exceptions/UncheckedIOException.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.core.exceptions;
+
+import java.io.IOException;
+
+/**
+ * Wraps a {@link IOException} in a {@link RuntimeException}.
+ * @author Sergio del Amo
+ * @since 3.5.6
+ */
+public class UncheckedIOException extends RuntimeException {
```

Comment:
Can we use https://docs.oracle.com/javase/8/docs/api/java/io/UncheckedIOException.html

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/8701/commits/1ab8748fba370bf20c03819d8e2085fbcbed1d93